WordPress/gutenberg
# Plugin: Remove PHP functions slated for 5.3 removal

This pull request seeks to remove PHP functions slated for removal in the 5.3 release:

https://wordpress.org/gutenberg/handbook/designers-developers/developers/backward-compatibility/deprecations/#5-3-0

- The PHP function `gutenberg_redirect_to_classic_editor_when_saving_posts` has been removed.
- The PHP function `gutenberg_revisions_link_to_editor` has been removed.
- The PHP function `gutenberg_remember_classic_editor_when_saving_posts` has been removed.
- The PHP function `gutenberg_can_edit_post_type` has been removed. Use [`use_block_editor_for_post_type`](https://developer.wordpress.org/reference/functions/use_block_editor_for_post_type/) instead.
- The PHP function `gutenberg_can_edit_post` has been removed. Use [`use_block_editor_for_post`](https://developer.wordpress.org/reference/functions/use_block_editor_for_post/) instead.

**Testing Instructions:**

Verify there are no errors in the load or general use of the editor.

Ensure that all referenced deprecated functions are no longer present in the codebase.
